### PR TITLE
Feature - Weather Gifs Website

### PR DESCRIPTION
**Pull request by carlosveigadev**
- Install webpack and tailwind;
- Add main HTML files;
- Create index.js file with main functions;
- Add function to fetch weather from API;
- Add function to manipulate data and show in the DOM;
- Add function to fetch Gifs;
- Add function to manipulate the gifs and show them in the DOM;
- Add toggle button to change from Celsius to Fahrenheit;
- Add Readme.

### CODE REVIEW

File: src/modules/changeTemperature.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function changeTemperature(data) {
```
Comment (mosaaleb) [inline src/modules/changeTemperature.js]: - [x] It uses ES6 syntax
. 

Let's make sure to convert all the functions to arrow function syntax.

File: src/modules/checkUvIndex.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+function checkUvIndex(index) {
```
Comment (mosaaleb) [inline src/modules/checkUvIndex.js]: - [x] let's replace it with arrow function syntax

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link href="./assets/css/tailwind.css" rel="stylesheet">
```
Comment (mosaaleb) [inline dist/index.html]: - [x] There is a console error coming from this line. I believe that you already installed tailwind using PostCSS as instructed [here](https://tailwindcss.com/docs/installation#installing-tailwind-css-as-a-post-css-plugin)? If that's right, we don't need this line. PostCSS already takes care of compiling tailwind styles.

Screenshot from 2020-12-12 22-21-17

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Well done @wrakc :clap:. You have handled most of the requirements :100:. Yet there are other requirements need to be handled before moving on.

- [x] **[OPTIONAL]** Let's add a  ‘loading’ component that displays from the time the form is submitted until the information comes back from the API. This will improve the experience of your app visitors.

- [x] **[OPTIONAL]** The page takes a few seconds before applying the styles. This issue comes from that the browser every time loads your app takes a very long time processing through your compiled final CSS file which contains all the styles that come from tailwind (around 3MB). To fix this issue you can use `purgecss` to remove all the un-used styles from the final build. Take a look at how to [optimize](https://tailwindcss.com/docs/optimizing-for-production) for production.


Please handle these requirements and submit a new code review.

##### Muhammad Ebeid

**Comment by carlosveigadev** [comment]
Hi @mosaaleb,
Thanks for the review. 
I implemented the improvements as suggested.

**Review by christianotieno — APPROVED** [review]
### Milestone Approved 🍾 
Hello @wrakc 👋 
 ##
This feature branch has met all the requirements needed for approval. 👏 
Congratulations on the hard work done on this project. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy coding 💻 
#### Christian Otieno (TSE)

**Pull request merged**